css/css-box/margin-trim/flex-block-end-trimmed-only-ref.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>margin-trim: flex-block-end-trimmed-only</title>
+<link rel="author" href="mailto:sammy.gill@apple.com">
+<link rel="help" href="https://drafts.csswg.org/css-box-4/#margin-trim-flex">
+<meta name="assert" content="block-end should only trim margin block end of flex items">
```

Comment:
nit: `s/only trim/trim only/` to match the wording-order in the testcase and in every other usage of this phrase in this patch.